swarm/api/api.go

```diff
@@ -987,6 +965,13 @@ func (a *API) ResolveFeed(ctx context.Context, uri *URI, values feed.Values) (*f
 	return fd, nil
 }
 
+// NewTag exposes chunk.Tags New method for incoming upload requests
```

Comment:
what is the purpose of this?

Reply:
middleware in the http api needs to create a new tag on `POST` requests. otherwise we can't access `chunk.Tags`